BossMod/Modules/Endwalker/HuntA/Gurangatch.cs

```diff
@@ -25,84 +25,44 @@ public enum AID : uint
         BoneShaker = 27512, // Boss->self, 4.0s cast, range 30 circle
     };
 
-    class Slammer : Components.GenericAOEs
+    class Slammer : Components.GenericRotatingAOE
     {
-        private int _remainingSlams = 0;
-        private Angle _slamDir;
-        private Angle _slamDirIncrement;
+        private Angle _increment;
```

Comment:
Do you actually need this field here? Looking at the code, it's always set in OnCastStart, then reset back immediately after - could just use a local variable?

Reply:
updated it